package.json

```diff
@@ -18,7 +18,7 @@
     "@dylan/coerce": "~0.2.0",
     "@dylan/engines": "github:shanebo/engines#0.1.0",
     "beard": "~0.8.1",
-    "mime": "2.3.1",
```

Comment:
Seems fitting to squash this commit with the commit where you actually utilize mrmime